config/environments/development.rb

```diff
@@ -23,7 +23,7 @@
   else
     config.action_controller.perform_caching = false
 
-    config.cache_store = :null_store
+    config.cache_store = :memory_store
```

Comment:
Let's make this an environment variable?

Reply:
We'll make this an environment variable once we introduce redis to this.